gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -19224,6 +19368,281 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  uicc_pathologic_t:
+    $ref:
+      - "_terms.yaml#/uicc_pathologic_t/common"
+    enum:
+      - T0
+      - T1
+      - T1a
+      - T1a1
+      - T1a2
+      - T1b
+      - T1b1
+      - T1b2
+      - T1c
+      - T1c2
+      - T1mi
+      - T2
+      - T2a
+      - T2a1
+      - T2a2
+      - T2b
+      - T2c
+      - T2d
+      - T3
+      - T3a
+      - T3b
+      - T3c
+      - T3d
+      - T4
+      - T4a
+      - T4b
+      - T4c
+      - T4d
+      - T4e
+      - TX
+      - Ta
+      - Tis
+      - Tis (DCIS)
+      - Tis (LCIS)
+      - Tis (Paget's)
+      - Unknown
+      - Not Reported
+      - Not Allowed To Collect
```

Comment:
if Not Allowed To Collect is deprecated in  ajcc_pathologic_t, do we need to include this here, or should it be left out from the new property?

Reply:
No need for enumDefs.  I removed all of the "Not Allowed to Collect" values from new properties.